src/components/OrgFile/components/AttributedString/index.js

```diff
@@ -11,26 +13,46 @@ import classNames from 'classnames';
 export default ({ parts, subPartDataAndHandlers }) => {
   let className;
 
+  let location = useLocation();
+
+  const renderLink = (part) => {
+    const id = part.get('id');
+    const uri = part.getIn(['contents', 'uri']);
+    const title = part.getIn(['contents', 'title']) || uri;
+    if (uri.startsWith("file:")) {
+      const target = uri.substr(5);
+      const isRelativeOrgFileLink =
+        !target.startsWith("/") &&
+        !target.startsWith("~") &&
+        !target.startsWith("../") &&
+        uri.endsWith(".org");
+      if (isRelativeOrgFileLink) {
+        const dir = location.pathname.match(/.*\//);
+        return (
+          <Link key={id} to={dir + target}>
```

Comment:
Do we need the `dir`? Relative (even `../`) would work without `dir` – at least in the `<a ...>`.